syft/file/license.go

```diff
@@ -21,7 +21,7 @@ type LicenseEvidence struct {
 func NewLicense(value string) License {
 	spdxExpression, err := license.ParseExpression(value)
 	if err != nil {
-		log.Trace("unable to parse license expression: %s, %w", value, err)
+		log.Tracef("unable to parse license expression: '%s', error: '%v'", value, err)
```

Comment:
nit
```suggestion
		log.WithFields("error", err).Tracef("unable to parse license expression: %q", value)
```

Reply:
This will generate **a lot** of errors (at least when I was programming this), as many license strings cannot be parsed.
Or has license parsing been 'fixed' in the mean time? I think this might be the underlying issue: #1964

Reply:
FWIW -- I think it's ok to be noisy, as long as we're using trace logging for it. In other words, these would only be displayed with -vvv not -vv.

Reply:
.. and I'm going to swap all the logging to use the `.WithFields`